packages/types/src/clerk.ts

```diff
@@ -13,8 +12,22 @@ export type UnsubscribeCallback = () => void;
 export type BeforeEmitCallback = (
   session: ActiveSessionResource | null,
 ) => void | Promise<any>;
+
 export type SignOutCallback = () => void | Promise<any>;
 
+export type SignOutOptions = {
```

Comment:
🙃 It would be great to align naming for method argument types. In same cases we use `FunctionXParams` in others `FunctionXOptions`. I prefer the first. WDYT?

Reply:
Completely agree - and we should try to align every other method  as well (already did that in #99 for many methods)

In this particular case, `signOut()` accepts 2 arguments: an optional `SignOutCallback` function and an optional `SignOutOptions` object so it follows this convention. `SignOutOptions` describes the 2nd argument, not the param list as a whole